reference-app/build.gradle

```diff
@@ -230,7 +230,7 @@ dependencies {
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
     def powerMockVersion = '2.0.7'
     implementation project(":opensrp-anc")
-    implementation('org.smartregister:opensrp-client-native-form:2.1.17-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-native-form:2.1.16-PREVIEW-SNAPSHOT@aar') {
```

Comment:
Why are we using a Preview version here? Why has the correct version not been published?